docs/integrations/azure/discovery.md

```diff
@@ -12,6 +12,34 @@ DevOps organization and register entities matching the configured path. This can
 be useful as an alternative to static locations or manually adding things to the
 catalog.
 
+## Installation
+
+You will have to add the processors in the catalog initialization code of your
+backend. They are not installed by default, therefore you have to add a
+dependency to `@backstage/plugin-catalog-backend-module-github` to your backend
```

Comment:
typo: github -> azure

Reply:
whoops, my turn for a copy & paste error. Thanks for catching.